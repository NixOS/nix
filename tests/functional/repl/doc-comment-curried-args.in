:l doc-comments.nix
:doc curriedArgs
x = curriedArgs 1        
"Note that users may not expect this to behave as it currently does"
:doc x
"This won't produce docs; no support for arbitrary values"
:doc x 2
