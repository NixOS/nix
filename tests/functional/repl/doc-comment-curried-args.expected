Nix <nix version>
Type :? for help.
Added <number omitted> variables.

Function curriedArgs
    … defined at
    /path/to/tests/functional/repl/doc-comments.nix:48:5

    A documented function.


"Note that users may not expect this to behave as it currently does"

Function curriedArgs
    … defined at
    /path/to/tests/functional/repl/doc-comments.nix:50:5

    The function returned by applying once

"This won't produce documentation, because we can't actually document arbitrary values"

error: value does not have documentation


